config/locales/en.yml

```diff
@@ -273,9 +273,19 @@ en:
   static_pages:
     not_authorised:
       title: You are not yet authorised to use the Teaching Jobs service
-      text:
-        - You can register your interest to use the service by emailing <a href='mailto:teachingjobs@digital.education.gov.uk', class='govuk-link'>teachingjobs@digital.education.gov.uk</a>. It is currently open to schools in Cambridgeshire and the North East but will be rolled out in phases to other areas across England.
+      without_email:
+        text:
+          - You can register your interest to use the service by emailing <a href='mailto:teachingjobs@digital.education.gov.uk', class='govuk-link'>teachingjobs@digital.education.gov.uk</a>. It is currently open to schools in Cambridgeshire and the North East but will be rolled out in phases to other areas across England.
 
-        - If your Teaching Jobs account has already been set up but you are unable to sign in, email the same address with “help” in the subject line.
+          - If your Teaching Jobs account has already been set up but you are unable to sign in, email the same address with "help" in the subject line.
+      with_email:
+        text:
+          - You've tried to sign in with %{email}, which is not associated with a Teaching Jobs account. If you have an account associated with another email, please first sign %{email} out of DfE Sign-in. Then sign into Teaching Jobs at <a href='https://profile.signin.education.gov.uk/' class='govuk-link'>https://profile.signin.education.gov.uk/</a> using your authorised email.
+
+          - If you don't have a Teaching Jobs account you will need to wait for an invitation for one. Invitations are being sent out gradually, by region. When you have accepted the invitation and created your account you will be able to start listing roles at your school.
+
+          - Read the Teaching Jobs <a href='https://dfedigital.blog.gov.uk/2018/09/21/how-were-rolling-out-our-search-and-listing-service-to-more-schools-to-support-their-teacher-recruitment-needs/' class='govuk-link'>blog on the service's phased roll-out</a> to learn when schools in your area will be invited.
```

Comment:
Can you configure the blog post link using a locale variable please. This should enable us to clearly see and manage any css and details of the link component in the view.